screenshot.spec.ts

```diff
@@ -0,0 +1,42 @@
+import * as fs from 'fs';
+import { test } from '@playwright/test';
+import { argosScreenshot } from '@argos-ci/playwright';
+import { extractSitemapPathnames, pathnameToArgosName } from './utils';
+
+// Constants
+const siteUrl = 'http://localhost:3000';
+const sitemapPath = './build/sitemap.xml';
+const stylesheetPath = './screenshot.css';
+const stylesheet = fs.readFileSync(stylesheetPath).toString();
+
+// Wait for hydration, requires Docusaurus v2.4.3+
+// Docusaurus adds a <html data-has-hydrated="true"> once hydrated
+// See https://github.com/facebook/docusaurus/pull/9256
+function waitForDocusaurusHydration() {
+  return document.documentElement.dataset.hasHydrated === 'true';
+}
+
+function screenshotPathname(pathname: string) {
+  test(`pathname ${pathname}`, async ({ page }) => {
+    const url = siteUrl + pathname;
+    await page.goto(url);
+    await page.waitForFunction(waitForDocusaurusHydration);
+    await page.addStyleTag({ content: stylesheet });
+    await argosScreenshot(page, pathnameToArgosName(pathname));
+  });
+}
+
+test.describe('Docusaurus site screenshots', () => {
+  const pathnames = extractSitemapPathnames(sitemapPath).filter((pathname) =>
+    pathname.match(
+      ' ^/$|' +
+        'mana-calculator|' +
+        'learn/glossary|' +
+        'build/networks-endpoints|' +
+        'create-mnemonic|' +
+        '(api|apis)/core',
```

Comment:
This results in quite a few redundant `api` matches. I guess we can just limit this to one page? Like just `/apis/core/v2/returns-the-health-of-the-node` which in principle will always be there, regardless of spec changes?

Reply:
I'm fine either way, it's just that if one thing changes in the API plugin, we need to approve multiple pages for the same change. But I also understand your point, so I'm fine merging it like it is so we can update to V3, and then I'm happy to create a dog feeding page or section in the near future so we don't have this problem. It's only a temporary inconvenience that way.